bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java

```diff
@@ -1042,19 +937,6 @@ public void setCornerRadius(int radius) {
 		parent.redraw();
 	}
 
-	@Override
-	public void setShadowVisible(boolean visible) {
```

Comment:
Isn't it API?

Reply:
Not API, package is x-internal/x-friend

Reply:
What's even more weird is that the interface implemented is in internal package so even technically it can't be API.